### ISSUE TITLE
2.1 Material Rotation Schedule

### ISSUE THREAD

**Issue by jttower** [gryphonshafer/Quizzing-Rule-Book#217]
At some point it would be nice to revise this section so the dates/material at the top are a more current year, since it hasn't been changed since 2020.  I don't think this needs to happen every year, but every 4-5 years seems good (which is where we are at at this point).  I think in the 2025 revision, it should be updated since this is an easy thing to do.

**Comment by ARMediting** [comment]
Is there a way we can either set this to update ahead of time, like approve a change now for a few years from now, or excuse this from the typical pull request process, since it's not a challengable section but just a preface?

**Comment by 24RMiller** [comment]
I agree that this would be a good year to make this change. I would like to think this could just be a simple, informal change that would not have to follow the protocols, as I don't there would be any objections.

**Comment by dickaum** [comment]
Yeah, since we aren't doing Mark anymore this is easy to make an automatic update.
